kickstarts/base.ks.erb

```diff
@@ -1,5 +1,8 @@
 # Kickstart file automatically generated by anaconda.
+
 # Run the text based installer to see dependency info in screenshots
+text
```

Comment:
@geertj Is this change still required?  See above note about VNC.  If we can't do VNC in text mode or the info we get in text mode VNC is less useful, I'd be against doing this.